screens/Home.js

```diff
@@ -19,29 +19,134 @@ import {
   Menu,
   MenuOptions,
   MenuOption,
-  MenuTrigger, renderers
+  MenuTrigger,
+  renderers
 } from "react-native-popup-menu";
 import getStyleSheet from "../styles/themestyles";
 import { ScreenStyles, FontStyles } from "../styles/global";
 import firebase from "react-native-firebase";
 import DatabaseManager from "../components/DatabaseManager";
 import { showMessage } from "react-native-flash-message";
+import AppIntroSlider from "react-native-app-intro-slider";
+
+const tutorialSlides = [
+  {
+    key: "slide1",
+    title: "Welcome to FitnessGo!",
+    text:
+      "This app allows you to create timers for your workouts, discover workouts from other people, and share your own workouts with your friends.",
```

Comment:
FitnessGo allows you to create custom workouts, discover workouts from other people, and share your workouts with your friends.